Frameworks/UIKit/UIApplication.mm

```diff
@@ -1636,6 +1640,18 @@ - (void)_sendNotificationReceivedEvent:(NSString*)notificationData {
     // todo-nithishm-05262016 - Implement UILocalNotification and call LocalNotification delegate here.
 }
 
+- (void)_sendVoiceCommandReceivedEvent:(WMSSpeechRecognitionResult*)voiceCommandResult {
+    if ([self.delegate respondsToSelector:@selector(application:didReceiveVoiceCommand:)]) {
```

Comment:
This needs to be defined in UIApplicationDelegate.h so apps can implement them. I seem to have missed this in my previous review